src/main/java/campus/infrastructure/Ui.java

```diff
@@ -0,0 +1,132 @@
+package campus.infrastructure;
+
+import campus.exceptions.CampusException;
+import campus.tasks.Task;
+
+/**
+ * Deals with interactions with the User
+ */
+public class Ui {
+    /**
+     * Produces a greeting message
+     */
+    public String printGreeting() {
+
+        String message = "------------------------------\n"
+                + "  ____\n"
+                + " / ___|__ _ _ __ ___  _ __  _   _ ___\n"
+                + "| |   / _` | '_ ` _ \\| '_ \\| | | / __|\n"
+                + "| |__| (_| | | | | | | |_) | |_| \\__ \\ \n"
+                + " \\____\\__,_|_| |_| |_| .__/ \\__,_|___/\n"
+                + "                     |_|\n"
+                + "\n"
+                + "Hello! I'm Campus\n"
+                + "What can I do for you?\n"
+                + "\n"
+                + "------------------------------\n";
+
+        return message;
+    }
+
+    /**
+     * Produce a goodbye/exit message
+     */
+    public String printExiting() {
+        String message = "------------------------------\n"
+                + "Bye. Hope to see you again soon!\n"
+                + "\n"
+                + "------------------------------\n";
+        return message;
+    }
+
+    /**
+     * Displays all the current items in the TaskList in a specified format with numbering
+     * @param taskList the object containing all the tasks in a list
+     */
+    public String printTaskList(TaskList taskList) {
```

Comment:
This may be a bad name for this method, given that it doesn't actually print it out, just return the output as a string. Maybe `formatTaskList`?

Reply:
hmm, this name has been changed before by another reviewer, I think i like `formatTaskList` than the current `printTaskList`, although this gives me problems for the other methods in which similarly, things are not being printed but returned as a string. `formatGreeting` and `formatExitting` both seem like strange names for the other methods following this suggestion...